src/arch/x86_64/kernel/vbe_console.c

```diff
@@ -27,6 +27,8 @@ static uint32_t brightcolors[] = {
   [VTCOLOR_CYAN] = 0xFFE0FFFF,  [VTCOLOR_GREY] = 0xFFFFFF,
 };
 
+multiboot_tag_framebuffer_common_t* multiboot_framebuffer_common = NULL;
```

Comment:
Why is this needed?

Reply:
so we could attach the framebuffer entry for the video_api